core/templates/dev/head/components/state/response_header_directive.html

```diff
@@ -80,9 +80,12 @@
     <em ng-if="getOutcome().newStateName">(<[getOutcome().newStateName]>)</em>
     <em ng-if="!getOutcome().newStateName">Nowhere yet...</em>
   </span>
-  <span ng-if="getOutcome() && (isOutcomeLooping() || (!isCorrect() && !getOutcome().dest && isInQuestionMode()))">
+  <span ng-if="getOutcome() && isOutcomeLooping() && !isInQuestionMode()">
     <em>(try again)</em>
   </span>
+  <span ng-if="isInQuestionMode() && getOutcome() && !isCorrect() && !getOutcome().dest">
```

Comment:
Would this lead to confusion if the return value of isCorrect() is in conflict with that of getOutcome().dest?

Reply:
There should ideally be some check which prevents the situation where an answer isn't correct when a destination is specified, and the other way round. (I think).

Reply:
Such a check isn't present, even for explorations i.e you can mark a default outcome also as correct. Should it be added? I'll remove the getOutcome().dest check here as correctness is all we are looking for at this point.

Reply:
I think it's technically possible for a default outcome to be correct (though that's a rather weak question). I think focusing only on correctness in this check seems sensible.

Reply:
Ok, done.